debian/control

```diff
@@ -30,6 +30,7 @@ Depends:
     libkdesk-dev,
     gir1.2-gtk-3.0 (>= 3.22.11),
     kano-settings (>= 1.3-1),
+    apt-transport-https,
```

Comment:
Let's add the min version of the package here `(>= 1.4.8)` so we know in the future what it used to be when it changes. I think it's good practice to always specify a min version.

Reply:
Good idea.